reference/import_export/formats.rst

```diff
@@ -24,3 +24,9 @@ CSV formats used for import/export profiles in a default Akeneo PIM installation
     formats/option
     formats/product
     formats/variant-group
+
+
+
+.. note::
+
+  XLSX format is also available. Data remain the same. You also have the possibility to export files with tabs.
```

Comment:
` You also have the possibility to export files with tabs`: are you sure it have been done ?